Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
     durabilityHint('PERFORMANCE_OPTIMIZED')
   }
   parameters {
-    string(name: 'ELASTIC_STACK_VERSION', defaultValue: "6.5", description: "Elastic Stack Git branch/tag to use")
+    string(name: 'ELASTIC_STACK_VERSION', defaultValue: "7.0.0-beta1", description: "Elastic Stack Git branch/tag to use")
```

Comment:
Afaic since this is not released yet you need to point to the correct docker images.